app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,5 @@
+@import "foundation_and_overrides";
+
+body {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.